tools/pl-paths.js

```diff
@@ -4,7 +4,7 @@
 
 const {
   patternPaths,
-} = require('../dist/pl/styleguide/data/patternlab-data.json');
+} = require('../dist/app-pl/pl/styleguide/data/patternlab-data.json');
```

Comment:
Wondering why we need to nest this deeply.  Why both "app-pl" and "pl" and not just "app-pl" itself?

Reply:
See structure of app output in dist/: there must be a pl folder to run pl as a sibling to assets.